tests/improver-weighted-blending/11-cycletime.bats

```diff
@@ -36,7 +36,8 @@
   KGO="weighted_blending/cycletime/kgo.nc"
 
   # Run weighted blending with linear weights and check it passes.
-  run improver weighted-blending 'forecast_reference_time' 'weighted_mean' --cycletime '20171129T0900Z'\
+  run improver weighted-blending 'forecast_reference_time' 'weighted_mean' \
+      --y0val 0.0 --ynval 4.0 --cycletime '20171129T0900Z'\
```

Comment:
I think the input of this test only has two forecast_reference_times in so these weighting give 100% to the later time and 0% to the earlier to the earlier forecast_reference time. This is fine as a test, but I just wanted to check whether this is what you meant to do?

Reply:
Good point, I hadn't realised.  I'd actually like to change this, will do that quickly and regenerate the data now.